wrt/wrt-apptools-android-tests/apptools/pkgName.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python 
+# coding=utf-8 
+import random,os,sys,unittest,allpairs 
+reload(sys) 
+sys.setdefaultencoding( "utf-8" ) 
+class TestCaseUnit(unittest.TestCase): 
+ 
+  def test_pkgName_positive1(self):
+     self.assertEqual("PASS", allpairs.tryRunApp("positive1", "crosswalk-app create org.xwalk.tests --crosswalk=/opt/wrt-apptools-android-tests/tools/crosswalk*.zip"))
```

Comment:
Better put these different pkgnames into a string array and make this assert a function,  that we can easy to read and update, also in order to reduce redundancy.